common/src/main/scala/explore/model/RootModel.scala

```diff
@@ -0,0 +1,20 @@
+package explore.model
+
+import cats.effect._
+
+import crystal._
+import monocle.macros.Lenses
+
+import scala.concurrent.ExecutionContext.global
+
+@Lenses
+case class RootModel(
+  target: Target
+)
+
+object AppState {
```

Comment:
I liked it when this was a trait mixed with `IOApp` why did it change?

Reply:
The challenge is accessing the same global state from the different projects. In this case, for example, `explore` and `conditions`. If the global state is in `explore`'s `Main`, `conditions` won't be able to see it since it's downstream.

We can do the same in `conditions`'s `Main`, but then we have 2 states.

I don't like that much declaring the implicits either instead of getting them from the `IOApp`. We could have a global initialization method that is called by all `Main`s and pass those implicits, but that means that global state is not created with the object storing it and we have to have a mutable placeholder for it.

Or maybe there's another solution I'm not seeing right now.